udata/templates/organization/display.html

```diff
@@ -15,11 +15,8 @@
 {% endblock %}
 
 {% block main_content %}
-<div itemscope itemtype="http://schema.org/{% if org.public_service %}Government{% endif %}Organization">
-    <meta itemprop="alternateName" content="{{ org.slug }}" />
-    <meta itemprop="logo" content="{{ org.logo(external=True) }}" />
-    <meta itemprop="url" content="{{ url_for('organizations.show', org=org, _external=True) }}" />
-    <h1 itemprop="name">
+<div>
```

Comment:
Need to check, but I think the extras divs can be safely removed as they where present only for microdata scoping